src/main/resources/ForumCode/Macros.xml

```diff
@@ -1036,5 +1036,136 @@ addthis:description="${escapetool.xml($addThisDesc)}"
   #end
 
 #end
+
+#**
+ * Edit document title
+ *#
+#macro(edittitle $errorkey)
+  #if($xcontext.action != 'view')
+    #set($discard = $xwiki.jsx.use("ForumCode.Macros"))
+    #set($titleErrorKey = "$!xcontext.macro.params.errorkey")
+    {{html clean="false"}}
+      &lt;!-- Validate form with Groovy --&gt;
+      &lt;input type="hidden" name="xvalidate" value="1" /&gt;
+      &lt;input type="hidden" name="xvalidation" value="ForumCode.ValidateForum" /&gt;
+    {{/html}}
+    #set ($titleIsValid = !$xcontext.validationStatus.errors.contains($titleErrorKey))
+    ##
+
+    {{html wiki="true"}}
+      &lt;dt class="field-mandatory"&gt;
+        &lt;label&gt;$services.localization.render('conversations.forum.title')&lt;/label&gt;
+        #if (!$titleIsValid)
+          &lt;span class="xErrorMsg"&gt;$services.localization.render($titleErrorKey)&lt;/span&gt;
+        #end
+      &lt;/dt&gt;
+      &lt;dd&gt;
+        #if("$!request.docTitle" != '')
+          #set($docTitle = $request.docTitle)
+        #else
+          #set($docTitle = $request.docTitle)
+        #end
```

Comment:
As mentioned on the original commit: https://github.com/xwiki-contrib/application-forum/commit/fe6b0d03ad9ef6c0065194b8232c481f767f3a74
This if/else does not make any sense.. having the same instructions in both branches.